src/sentry/api/urls.py

```diff
@@ -2100,6 +2101,11 @@
         ProjectReleasesEndpoint.as_view(),
         name="sentry-api-0-project-releases",
     ),
+    re_path(
+        r"^(?P<organization_slug>[^\/]+)/(?P<project_slug>[^\/]+)/releases/thresholds/$",
+        ReleaseThresholdEndpoint.as_view(),
```

Comment:
I imagine one day we'll have a `threshold details` api as well.

Maybe we call this class the `ReleaseThresholdIndex` instead? (do they all need to be called `...Endpoint`?)
Index post being the generic "create a trheshold", or maybe even can update to pass a list of thresholds to create
Details being to modify a specific threshold (PUT/DELETE/GET)

Reply:
Ah that's fair 👍